extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-sidebar-1.0.css';
+import {Layout} from '../../../src/layout';
+
+export class AmpSidebar extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return layout == Layout.NODISPLAY;
+  }
+
+  /** @override */
+  buildCallback() {
+
+  }
+}
+
+AMP.registerElement('amp-sidebar', AmpSidebar, CSS);
```

Comment:
declare version '1.0' here for future testing purpose. 
https://github.com/ampproject/amphtml/blob/master/extensions/amp-sticky-ad/1.0/amp-sticky-ad.js#L256